### ISSUE TITLE
Wrong output format

### ISSUE THREAD

**Issue by STayinloves** [kyzhouhzau/BERT-NER#51]
The `conlleval.pl` script accepts input file format like `<span> <groundtruth> <prediction>`, but the model's output is `<span> <prediction> <groundtruth>`. Although it doesn't affect the F1 score, it swaps the values of precision and recall.

**Comment by kyzhouhzau** [comment]
Yes, you are right thank you for your help for this hidden problem. I will update it.
